scripts/custom/clean_hour.py

```diff
@@ -0,0 +1,18 @@
+from scripts import utils
+
+
+def clean_hour(raw_input):
+    """Convert 800, 1400, 1715 to 08:00, 14:00, 17.15
+    """
+    if not isinstance(raw_input, str):
+        raw_input = str(raw_input)
+    if utils.is_empty(raw_input):
+        return ""
+
+    hour = raw_input
+    while len(hour) < 4:
```

Comment:
there is a function on string to do that: https://www.tutorialspoint.com/python/string_rjust.htm

Reply:
so cool this function